src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -650,6 +657,9 @@ private void ApplyAutoEvoRun()
         HistoryListMenuIndexChanged(currentGeneration);
 
         currentGenerationLabel.Text = currentGeneration.ToString();
+
+        totalTimeUsed += autoEvoRun.RunDuration;
+        totalTimeUsedLabel.Text = totalTimeUsed.ToString("g");
```

Comment:
Shouldn't this use `CurrentCulture` to make sure the time is formatted according to the player's language preference?